bindings/nodejs/tests/setup.js

```diff
@@ -56,6 +57,8 @@ QUnit.config.callbacks.log.push( function( status ) {
 
 	// Parameters: status: { module, result(t/f), message, actual, expected, testId, runtime }
 
+	results.getTestResult( status, success, failure );
```

Comment:
Here it appears that `success` and `failure` will have their value changed by this call. Is this right?